Версия № 7:

ZotovSlava opened the pull request:
1) Настроена возможность доступ к методам программы через HTTP API; 2) Добавлены тесты обработчиков запросов.

ZotovSlava committed:
Версия № 7:
1) Настроена возможность доступ к методам программы через HTTP API;
2) Добавлены тесты обработчиков запросов.

ZotovSlava committed:
Версия № 7.1:
1) Исп. GitTest

ZotovSlava committed:
Версия № 7.1:
1) Исп. GitTest

ZotovSlava committed:
Версия № 7.1:
1) Исп. GitTest

ZotovSlava committed:
Версия № 7.1:
1) Исп. GitTest

ZotovSlava committed:
Версия № 7.1:
1) Исп. GitTest

litvitnik commented on src/BaseHttpHandler.java:
```diff
@@ -0,0 +1,34 @@
+        h.close();
+    }
+
+    protected void sendHasInteractions(HttpExchange h) throws IOException {
```
Пересечения - это Intersection, а Interactions - это взаимодействия

ZotovSlava committed:
Версия № 7.3:
1) В class BaseHttpHandler изменено название метода на более корректное.
(sendHasInteractions -> sendHasIntersection)
